deploy/complete/helm-chart/setup/values.yaml

```diff
@@ -123,31 +123,35 @@ grafana:
     - grafana-piechart-panel
     - btplc-status-dot-panel
 
-# https://github.com/helm/charts/blob/master/stable/jenkins/README.md
+# https://github.com/jenkinsci/helm-charts/blob/main/charts/jenkins/README.md
 jenkins:
   enabled: false
-  master:
+  #https://github.com/jenkinsci/helm-charts/blob/main/charts/jenkins/values.yaml
+  controller:
     serviceType: ClusterIP
     jenkinsUriPrefix: "/jenkins"
     ingress:
       # enable ingress only if the ingress-controller `nginx-ingress` is enabled.
       enabled: true
-      paths:
-        - path: /jenkins
-          backend:
-            serviceName: >-
-              {{ template "jenkins.fullname" . }}
-            servicePort: 8080
-      apiVersion: "networking.k8s.io/v1beta1"
+      apiVersion: networking.k8s.io/v1
+      #apiVersion: "networking.k8s.io/v1beta1"
       labels: {}
-      annotations:
-        kubernetes.io/ingress.class: nginx
```

Comment:
The annotations: 
```
    kubernetes.io/ingress.class: nginx
    nginx.ingress.kubernetes.io/rewrite-target: /$2
```
are necessary to get the right ingress controller. The second annotations may be needed to make sure that everything after /jenkins works. You need to test.